codec-http2/src/test/java/io/netty5/handler/codec/http2/Http2MultiplexTransportTest.java

```diff
@@ -371,7 +374,7 @@ protected void initChannel(Channel ch) {
             bs.handler(new ChannelInitializer<Channel>() {
                 @Override
                 protected void initChannel(Channel ch) {
-                    ch.pipeline().addLast(clientCtx.newHandler(ch.alloc()));
+                    ch.pipeline().addLast(clientCtx.newHandler(ch.bufferAllocator()));
```

Comment:
Why we don't need the `BufferConversionHandler.bufferToByteBuf()` here but we need it above ?

Reply:
Good question! Turns out in the case above, the `SslHandler` receives a `write()` call _before_ `channelActive` is called, which is the decision to implicitly convert buffers or not is made. Perhaps this implicit conversion should either always happen, or never happen. Dynamic detection doesn't look so reliable.

Reply:
Let's remove the implicit conversion in a separate PR.

Reply:
PR: https://github.com/netty/netty/pull/12297